bowtie/_report.py

```diff
@@ -330,14 +342,26 @@ def generate_badges(self, target_dir: Path, dialect: str):
             impl_dir = target_dir / f"{lang}-{name}"
             impl_dir.mkdir(parents=True, exist_ok=True)
             r, g, b = 100 - int(pct), int(pct), 0
-            badge = {
+            badgePerDraft = {
                 "schemaVersion": 1,
                 "label": label,
                 "message": "%d%% Passing" % int(pct),
                 "color": f"{r:02x}{g:02x}{b:02x}",
             }
-            badge_path = impl_dir / f"{label.replace(' ', '_')}.json"
-            badge_path.write_text(json.dumps(badge))
+            badgeSupportedDraft = {
+                "schemaVersion": 1,
+                "label": "JSON Schema Versions",
+                "message": SupportedDrafts,
+                "color": "lightgreen",
+            }
+            badge_path1 = (
+                impl_dir / f"{label.replace(' ', '_')}_perDraftCompliance.json"
```

Comment:
We wanted to put these in folders rather than suffixes, so let's introduce separate directories for the two kinds of badges (one probably called `compliance` and the other `supported-versions`).